services/QuillLMS/app/models/sales_form_submission.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class SalesFormSubmission < ApplicationRecord
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true
```

Comment:
What about adding some kind of email validation [similar](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/models/user.rb#L141) to our User model: